api/v1/chyt_types.go

```diff
@@ -40,8 +40,12 @@ type ChytSpec struct {
 
 	Ytsaurus *corev1.LocalObjectReference `json:"ytsaurus,omitempty"`
 	Image    string                       `json:"image,omitempty"`
+	// Mark specified image as default for cliques.
 	//+kubebuilder:default:=false
 	MakeDefault bool `json:"makeDefault"`
+	// Create ch_public clique, which is used by default when running CHYT queries.
+	//+kubebuilder:default:=false
+	CreatePublicClique bool `json:"createPublicClique"`
```

Comment:
I suggest we make it `*bool` with `,omitempty` tag, so it would be possible to serialize spec to yaml/json and distinguish not set value from true/false. 
And I think the default value better be controlled on the code level, not spec level. That way we can change it if we want for users, who didn't set it, and if we have simple bool — all users will have false in their resource after the operator update.

I know that most other booleans in our spec don't work that way, but I think `*bool` is more flexible here, so new fields better be added that way.

Reply:
Sure.